includes/Modules/Ads.php

```diff
@@ -85,7 +93,64 @@ protected function setup_assets() {
 					),
 				)
 			),
+			new Script_Data(
+				'googlesitekit-ads-pax-config',
+				array(
+					'global'        => '_googlesitekitPAXConfig',
+					'data_callback' => function() {
+						if ( ! current_user_can( Permissions::VIEW_DASHBOARD ) ) {
+							return array();
+						}
+
+						return array(
+							'authAccess'      => array(
+								'oauthTokenAccess' => array(
+									'token' => (string) $this->authentication->get_oauth_client()->get_access_token(),
+								),
+							),
+							'locale'          => mb_substr( $this->context->get_locale( 'user' ), 0, 2 ),
```

Comment:
Is the `mb_` variant necessary? I think locale identifiers can only be in latin characters, no?

Reply:
It was just an extra precaution, but yeah went through the list of locales WP uses, and they are all in latin characters.